hello_vsi/FVP_Corstone_SSE-300.cfg

```diff
@@ -0,0 +1,17 @@
+# Notes:
+# 1) Disable visualization for CI environments.
+# 2) Disable gatting logic for memory
+# 3) Set stack_base and stack_limit to 0x0 
+#    according to: https://developer.arm.com/documentation/ka005824/latest
+#
+# Parameters:
+# instance.parameter=value                            #(type, mode)         default = 'def value'  : description : [min..max]
+#---------------------------------------------------------------------------------------------------
+mps3_board.visualisation.disable-visualisation=1      # (bool  , init-time) default = '0'          : Enable/disable visualisation
+mps3_board.DISABLE_GATING=1                           # (bool  , init-time) default = '0'          : Disable Memory gating logic
+cpu0.semihosting-stack_base=0x0                       # (int   , init-time) default = '0x20800000' : Virtual address of base of descending stack

```

Comment:
I am surprised that the different platforms require different parameters to be included.

Reply:
Aligned to minimum required differences.
i.e. FVP_Corstone_SSE-315 is using mps4_board instead of  mps3_board and requires also to disable the HLCD option.